widgets/FileCopyToGCSAction-action.json

```diff
@@ -0,0 +1,105 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "display-name" : "Copy File to GCS",
+  "configuration-groups": [
+    {
+      "label": "Properties",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Source Path",
+          "name": "sourcePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "project",
+          "widget-attributes" : {
+            "default": "auto-detect"
+          }
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Destination Bucket",
+          "name": "destPath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Wildcard",
+          "name": "fileRegex"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Number of parallel tasks",
+          "name": "numThreads"
+        }
+      ]
+    },
+    {
+      "label" : "Credentials",
```

Comment:
I think we should keep this simple and just use the autodetect option.

If we set the project using gcloud apis then we don't need to supply a separate file.

Reply:
removed